docs/guide/data/idraw-readme.txt

```diff
@@ -0,0 +1,459 @@
+                    (after) other graphics.
+
+     Send To Back   Send the selected graphics to the back of the
+                    drawing   so   that  they  are  drawn  behind
+                    (before) other graphics.
+
+     The Font menu contains a set of fonts in  which  to  display
+     text.  When you set the current font from the menu, you will
+     also set all the selected graphics' fonts to that  font.   A
+     font  indicator  in  the  upper  right  corner  displays the
+     current font.
+
+     The Brush menu contains a set of brushes with which to  draw
+     lines.   When  you  set the current brush from the menu, you
+     will also set all the selected  graphics'  brushes  to  that
+     brush.   The  nonexistent  brush  draws  invisible lines and
+     non-outlined graphics.  The arrowhead brushes add arrowheads
+     to  either  or  both  ends  of  lines,  multilines, and open
+     splines. A brush indicator in the upper left corner displays
+     the current brush.
+
+     The Pattern menu contains a set of patterns  with  which  to
+     fill  graphics but not text.  Text always appears solid, but
+     you can use a different color than black to get a  halftoned
+     shade.   When you set the current pattern from the menu, you
+     will also set all the selected graphics'  patterns  to  that
+     pattern.   The  nonexistent pattern draws unfilled graphics,
+     while the other patterns draw graphics filled with a  bitmap
+     or a halftoned shade.
+
+     The FgColor and BgColor menus contains a set of colors  with
+     which  to  draw graphics and text.  When you set the current
+     foreground or background color from the FgColor  or  BgColor
+     menu,  you  will  also  set all the selected graphics' fore-
+     ground or background colors.  The ``on'' bits in the bitmaps
+     for  dashed lines and fill patterns appear in the foreground
+     color while the ``off'' bits appear in the background color.
+     A  black  and  white printer will print a halftoned shade of
+     gray for any color other than black or  white.   The  brush,
+     pattern, and font indicators all reflect the current colors.
+
+     The Align menu contains  commands  to  align  graphics  with
+     other  graphics.   The  first  graphic  selected stays fixed
+     while the  other  graphics  move  in  the  order  they  were
+     selected  according  to  the  type of alignment chosen.  The
+     last Align command, Align to Grid, aligns  a  key  point  on
+     each  selected  graphic to the nearest point on idraw's grid
+     (see below).
+
+     The View menu contains the following commands:
+
+     New View       Create a duplicate idraw window containing  a
+                    second  view  of  the  current  drawing.  The
+                    second view may be panned, zoomed, and edited
+                    independently  of  the  first.  Any number of
+                    additional views may be made in this  manner.
```

Comment:
does this work as a table?

Reply:
same here. I don't know how to determine whether or not it works as a table